### DISCUSSION TITLE
503 Error: It stops and does not download.> is this the API down issue?

### DISCUSSION THREAD

**Discussion by alangihub** [davetapley/helium-tax#36] [General]
is this the HELIUM API down issue? It stops after 200  and does not download.   I tried a few hotspot IDs.   I Tried it in a new browser, and  rebooted the PC.  it had been working fine last week.  ( by the way...   great tool,  THANK YOU DAVE! )   

The  503 error I see in developer tools is "https://davetapley.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource."  

My ID was edited in below info: 

VM6:1 GET https://api.helium.io/v1/hotspots/11Ybc-MYI_IDL/rewards?cursor=eyJ0eG4iOiJJNW41QnhVNG9IeVJpV25PZTg4OTZIZG1xeHZNUldpUGFEZ0xoSTV1RGc4IiwiZW5kX2Jsb2NrIjo2NTUyMTgsImJsb2NrIjoxMDExNzc1LCJhbmNob3JfYmxvY2siOjEwOTg2MDB9 net::ERR_FAILED 503
(anonymous) @ VM6:1
dispatchXhrRequest @ bundle.js:2168
xhrAdapter @ bundle.js:2004
dispatchRequest @ bundle.js:2562
Promise.then (async)
request @ bundle.js:2378
wrap @ bundle.js:2884
(anonymous) @ bundle.js:16061
Promise.then (async)
o @ bundle.js:16061
Promise.then (async)
request @ bundle.js:2378
wrap @ bundle.js:2884
(anonymous) @ bundle.js:16061
Promise.then (async)
o @ bundle.js:16061
Promise.then (async)
request @ bundle.js:2378
wrap @ bundle.js:2884
(anonymous) @ bundle.js:16061
Promise.then (async)
o @ bundle.js:16061
Promise.then (async)
request @ bundle.js:2378
wrap @ bundle.js:2884
(anonymous) @ bundle.js:16061
Promise.then (async)
o @ bundle.js:16061
Promise.then (async)
request @ bundle.js:2378
Axios.<computed> @ bundle.js:2393
wrap @ bundle.js:2884
get @ bundle.js:436
list @ bundle.js:1739
nextPage @ bundle.js:507
hotspotTaxes @ bundle.js:16274
async function (async)
hotspotTaxes @ bundle.js:16228
addressTaxes @ bundle.js:16202
async function (async)
addressTaxes @ bundle.js:16201
(anonymous) @ bundle.js:41
davetapley.com/:1 Access to XMLHttpRequest at 'https://api.helium.io/v1/hotspots/11Ybc-MYI_IDL/rewards?cursor=eyJ0eG4iOiJJNW41QnhVNG9IeVJpV25PZTg4OTZIZG1xeHZNUldpUGFEZ0xoSTV1RGc4IiwiZW5kX2Jsb2NrIjo2NTUyMTgsImJsb2NrIjoxMDExNzc1LCJhbmNob3JfYmxvY2siOjEwOTg2MDB9' from origin 'https://davetapley.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by davetapley** [comment]
Thanks for reporting @alangihub.

I _believe_ this is caused by ongoing outage (per https://status.helium.com/), so 🤞🏻 it'll resolve itself when that does.

I've also got https://github.com/davetapley/helium-tax/issues/26 in the pipeline to better detect/show when this happens.
